lms/djangoapps/course_api/blocks/transformers/tests/test_milestones.py

```diff
@@ -0,0 +1,198 @@
+"""
+Tests for ProctoredExamTransformer.
+"""
+from mock import patch, Mock
+from nose.plugins.attrib import attr
+
+import ddt
+from gating import api as lms_gating_api
+from lms.djangoapps.course_blocks.transformers.tests.helpers import CourseStructureTestCase
+from opaque_keys.edx.keys import UsageKey
+from student.tests.factories import CourseEnrollmentFactory
+
+from ..milestones import MilestonesTransformer
+from ...api import get_course_blocks
+from openedx.core.lib.gating import api as gating_api
+from milestones.tests.utils import MilestonesTestCaseMixin
+import courseware
```

Comment:
Nit: For readability, we've been organizing our imports by (1) distance from current project (far->least) and then (2) alphabetically.